addons/analytic/models/analytic_account.py

```diff
@@ -174,3 +174,14 @@ def convert(amount, from_currency):
                 account.debit = -data_debit.get(account.id, 0.0)
                 account.credit = data_credit.get(account.id, 0.0)
                 account.balance = account.credit - account.debit
+
+    @api.ondelete(at_uninstall=False)
+    def _unlink_except_analytic_items(self):
+        """ Check if analytic items (e.g. account moves, analytic simulations) are associated to the account """
+        domain = [('company_id', 'in', [False] + self.env.companies.ids)]
```

Comment:
Why the domain on the company? Is it ok if the account is somehow used in other companies? Especially since you look at the companies in the environment/context.

And what's up with the `company_id=False`? It's a required field, no?